src/styles/variables.scss

```diff
@@ -0,0 +1,31 @@
+$clock-shadow: 0 0 50px 10px rgba(0, 255, 255, 0.6),
+  0 0 30px 5px rgba(0, 255, 255, 0.6),
+  0 0 10px 5px rgba(0, 255, 255, 0.6);
+$clock-hand-hour-shadow: 0 0 5px #ff105e,
+  0 0 10px #ff105e,
+  0 0 20px #ff105e,
+  0 0 30px #ff105e,
+  0 0 40px #ff105e,
+  0 0 55px #ff105e,
+  0 0 75px #ff105e;
+$clock-hand-minute-shadow: 0 0 10px #fff,
+  0 0 20px #fff,
+  0 0 30px #fff,
+  0 0 40px #0ff,
+  0 0 70px #0ff,
+  0 0 80px #0ff,
+  0 0 100px #0ff,
+  0 0 150px #0ff;
+$clock-hand-secunde-shadow: 0 0 10px #1bcdc4,
```

Comment:
Typo in variable name: Change $clock-hand-secunde-shadow to $clock-hand-second-shadow